control/servo_interface.cc

```diff
@@ -20,5 +22,22 @@ SetServoMessage create_servo_command(const QuadraframeStatus& qframe_status) {
   return servo_msg;
 }
 
+QuadraframeStatus create_quadraframe_status(const SetServoMessage& servo_msg) {
+  assert(servo_msg.servo_indices.size() == 4u);
+  assert(servo_msg.target_radians.size() == 4u);
+
+  for (std::size_t k = 0; k < servo_msg.servo_indices.size(); ++k) {
+    // Assert we have 0...3 and only 0...3
+    assert(servo_msg.servo_indices.at(k) == static_cast<int>(k));
```

Comment:
I could create packaging/unpackaging methods that would take vectors of structs to make it not possible to mismatch indices and targets so easily.